### DISCUSSION TITLE
Для нерабочего времени реализовать возможность И проигрования файла, И перевод на номер (голосовой почты)

### DISCUSSION THREAD

**Discussion by matveevandrey** [mikopbx/Core#442] [Ideas (Идеи)]
Очень логично перед переводом входящего звонка на голосовую почту м (по нерабочему времени) сначала проиграть приветствие/пояснение и только после этого перевести звонок на голосовую почту

СПасибо

**Comment by jorikfon** [comment]
Это решается переводом на IVR меню, в котором это все можно реализовать уже сейчас.

**Reply by Realmagnum** [reply]
Тоже делал именно так.

**Reply by matveevandrey** [reply]
> Это решается переводом на IVR меню, в котором это все можно реализовать уже сейчас.

спасибо
